### ISSUE TITLE
Crate a Project for the Support Dogecoin resource

### ISSUE THREAD

**Issue by zaridan** [dogecoin/dogecoin.com#143]
I have questions that I'd like to seek input on, and would love to have an opportunity to outline needs, tasks, etc, for getting this part of the site done.

Can we create a Project for this?

**Comment by patricklodder** [comment]
Of course! What do you have in mind? Would you like to work from question to answer or from subject to implementation?

**Comment by zaridan** [comment]
Probably from subject to implementation... but I've not set up nor used GitHub projects much, so that's a guesstimate based on assumptions :)

**Comment by zaridan** [comment]
... and to broaden the project, maybe we could make it for the whole of the Ðogecoin.com site rather than only for the Resources section?

**Comment by patricklodder** [comment]
Let's start with a well defined target, see if this works, then we can run other, bigger projects that build upon success... saves scope creep and overcommitment. 

So the project is up at https://github.com/dogecoin/dogecoin.com/projects/1 and we can start adding issues and pull requests to that.

**Comment by zaridan** [comment]
Sounds good, thanks @patricklodder !
